build-tests/api-documenter-test/etc/yaml/api-documenter-test.yml

```diff
@@ -31,6 +31,7 @@ functions:
     package: api-documenter-test!
     summary: An exported function with hyperlinked parameters and return value.
     remarks: ''
+    example: []

```

Comment:
I have updated your branch with the build test outputs.  (These files are supposed to be committed, but we don't have any validation for that, so you just have to remember to do it manually.)

Reply:
Thank you!